app/src/components/HomePage/ActionCards.tsx

```diff
@@ -0,0 +1,74 @@
+import type { TFunction } from "i18next";
+import { Box, Card, CardBody, CardHeader, Text } from "@chakra-ui/react";
+import NextLink from "next/link";
+import { MdOutlineAddchart } from "react-icons/md";
+import { Trans } from "react-i18next/TransWithoutContext";
+import DownloadButton from "@/components/HomePage/DownloadButton";
+import type { CityAttributes } from "@/models/City";
+import type { InventoryAttributes } from "@/models/Inventory";
+
+export function ActionCards(props: {
+  inventoryId: string | null;
+  t: TFunction;
+  city?: CityAttributes;
+  lng: string;
+  inventory?: InventoryAttributes;
+}) {
+  return (
+    <Box className="flex gap-[24px] ">
+      <NextLink
+        data-testid={"add-data-to-inventory-card"}
+        href={`/${props.inventoryId}/data`}
+      >
+        <Card
+          shadow="2dp"
+          backgroundColor="base.light"
+          borderColor="interactive.accent"
+          borderWidth="thin"
+          className="h-[132px] hover:shadow-xl"
+          py={0}
+          px={6}
+        >
+          <Box className="flex items-center w-fill">
+            <Box>
+              <Box className="flex items-center justify-center h-[48px] w-[48px] rounded-full bg-[#008600]">
```

Comment:
Would be good not to hardcode the color with Tailwind here but either use a Tailwind theme variable or even better use a color from the theme (since Box is a ChakraUI component that should be doable without much effort).